metomi/rose/metadata_gen.py

```diff
@@ -141,8 +141,31 @@ def type_gen(value):
 
 
 def main():
-    opt_parser = metomi.rose.opt_parse.RoseOptionParser()
+    opt_parser = metomi.rose.opt_parse.RoseOptionParser(
+        usage='rose metadata-gen [OPTIONS] [PROPERTY=VALUE ...]',
+        description='''
+Automatically generate metadata from an application or suite
+configuration. An aid for metadata development.
+
+WARNING: May Contain Thorns.
```

Comment:
_really ?_

Reply:
This command attempts to guess the type of inputs. It doesn't always give the right answer.